src/components/TodoList.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+ 
+    
+            <!-- <div >
```

Comment:
dont commit commented code. Shouldn't this code be used to add the strike-through functionality to the todo list?